hack/lib/golang.sh

```diff
@@ -482,6 +482,7 @@ EOF
   local go_version
   IFS=" " read -ra go_version <<< "$(GOFLAGS='' go version)"
   local minimum_go_version
+  # TODO(dims): Need to switch this to 1.19 once we update images to newer go version
```

Comment:
once this merges and CI switches to 1.19rc2, will people with a local go1.18 be generating/formatting changes that will fail CI?

Reply:
Yes, there is one file that has trouble, this is what happens when i run update-gofmt.sh using golang 1.18 (with this branch)

```
diff --git staging/src/k8s.io/apiserver/pkg/util/flowcontrol/patch_test.go staging/src/k8s.io/apiserver/pkg/util/flowcontrol/patch_test.go
index a5b01802cb95..b468b6d3b0d1 100644
--- staging/src/k8s.io/apiserver/pkg/util/flowcontrol/patch_test.go
+++ staging/src/k8s.io/apiserver/pkg/util/flowcontrol/patch_test.go
@@ -59,10 +59,10 @@ func Test_configController_generatePatchBytes(t *testing.T) {
                {
                        name: "check when message has a whitespace character that can be escaped",
                        condition: flowcontrol.FlowSchemaCondition{
-                               Type:               flowcontrol.FlowSchemaConditionDangling,
-                               Status:             flowcontrol.ConditionTrue,
-                               Reason:             "test reason",
-                               Message:            `test               none`,
+                               Type:   flowcontrol.FlowSchemaConditionDangling,
+                               Status: flowcontrol.ConditionTrue,
+                               Reason: "test reason",
+                               Message: `test          none`,
                                LastTransitionTime: metav1.NewTime(now),
                        },
                        want: []byte(fmt.Sprintf(`{"status":{"conditions":[{"type":"Dangling","status":"True","lastTransitionTime":"%s","reason":"test reason","message":"test \t\tnone"}]}}`, now.Format(time.RFC3339))),

```

The `Message` has a horizontal tab (hex 0x09) in the string and how that is handled is different between the go versions

Reply:
interesting... 1.19 gofmt and 1.18 gofmt dueling seems like a bug we should report... my understanding is that gofmt tries to only make changes a previous version would leave alone

as a workaround for us, you could put a blank line before and after the `Message` line and I bet both go1.19 and go1.18 would format it identically

Reply:
> interesting... 1.19 gofmt and 1.18 gofmt dueling seems like a bug we should report

well, looks intentional, given https://github.com/golang/go/issues/51910 / https://github.com/golang/go/commit/bf68170c638e7e69bedcc64fadfd83354fd06c10

Reply:
the workaround to avoid adding formatting friction to ~everyone's PRs the week of code freeze is probably worthwhile

Reply:
`blank line before and after the Message line` did not help @liggitt 

Can i switch this to "test  \u0009none" (2 spaces and then a tab are in between `"test"` and `"one"`

Reply:
> Can i switch this to "test \u0009none" (2 spaces and then a tab are in between "test" and "one"

yup, that'd work too

Reply:
Done!